core/src/main/java/nl/nn/adapterframework/parameters/ParameterList.java

```diff
@@ -32,30 +33,42 @@
  * @author Gerrit van Brakel
  */
 public class ParameterList extends ArrayList<Parameter> {
```

Comment:
Moeten we er niet toch een LinkedHashMap van maken?

Reply:
Ik zie nog niet direct een reden hiertoe.

Reply:
omdat contains() nu werkt met find, die itereert over alle waarden

Reply:
Contains override niet de default

Reply:
maar het is wel inefficient, of zijn er nooit zoveel parameters dat dat een issue wordt?